platform_secrets/service.py

```diff
@@ -36,16 +35,23 @@ class Service:
     def __init__(self, kube_client: KubeClient) -> None:
         self._kube_client = kube_client
 
-    def _get_kube_secret_name(self, project_name: str, org_name: Optional[str]) -> str:
+    def _get_kube_secret_name(
+        self,
+        project_name: str,
+        org_name: Optional[str],
+        *,
+        create_legacy_name: bool = False,
+    ) -> str:
         path = project_name
         if org_name:
             path = f"{org_name}/{path}"
-        return f"user--{path.replace('/', '--')}--secrets"
+        prefix = "user" if create_legacy_name else "project"
```

Comment:
I am trying to understand why we can't use `user--{project_name}--secrets` without the new prefix `project`?
we created projects for all existing users.

Reply:
We can use. I just decided to change secret name format since secrets don't belong to user anymore, they belong to the whole project.

Reply:
We can use. I just decided to change secret name format since secrets don't belong to user anymore, they belong to the whole project.

Reply:
I understand. since we don't migrate existing secrets to the new name format, we'll end up with the formats anyways. but that's fine I guess.